lib/cpp/src/thrift/server/TSimpleServer.cpp

```diff
@@ -88,7 +88,7 @@ void TSimpleServer::serve() {
       string errStr = string("Some kind of accept exception: ") + tx.what();
       GlobalOutput(errStr.c_str());
       continue;
-    } catch (string s) {
+    } catch (const string& s) {
```

Comment:
I'm curious, is there anything actually throwing a bare string?  I don't see anything, and in general concerned about the inability to debug errors caused by catching too much stuff.